### ISSUE TITLE
mv: can't rename '/tmp/backups/*.' caused by disabling checksums

### ISSUE THREAD

**Issue by ToshY** [tiredofit/docker-db-backup#243]
### Summary

Logging shows the message `mv: can't rename '/tmp/backups/*.': No such file or directory`:

```shell
dsmr-reader-docker-compose-postgres-backup-1  | 2023-08-22T20:34:24.888376407Z 2023-08-22.20:34:24 [NOTICE] ** [db-backup] Dumping PostgresSQL database: 'dsmr' and compressing with 'gzip'
dsmr-reader-docker-compose-postgres-backup-1  | 2023-08-22T20:34:26.131319520Z 2023-08-22.20:34:26 [INFO] ** [db-backup] DB Backup of 'pgsql_dsmr_postgres_20230822-203424.sql.gz' completed successfully
dsmr-reader-docker-compose-postgres-backup-1  | 2023-08-22T20:34:26.168926429Z 2023-08-22.20:34:26 [NOTICE] ** [db-backup] Backup of pgsql_dsmr_postgres_20230822-203424.sql.gz created with the size of 277131 bytes
dsmr-reader-docker-compose-postgres-backup-1  | 2023-08-22T20:34:26.203726216Z mv: can't rename '/tmp/backups/*.': No such file or directory
```

* This occurs when `ENABLE_CHECKSUM="false"`.
* This does not affect the actual backup creation, as the script still continues without any further issues, therefor it's only a minor bug.

### Steps to reproduce

Use `ENABLE_CHECKSUM="false"`.

### What is the expected *correct* behavior?

No (re)moving of checksums when the `ENABLE_CHECKSUM` is set to `false` 

Probably adding a `if var_true "${ENABLE_CHECKSUM}" ; then` would suffice.

### Environment

Image version / tag: `3.9.9`
Host OS: `Raspbian GNU/Linux 11 (bullseye); Model: Raspberry Pi 3 Model B`

<details>
<summary>Any logs | docker-compose.yml</summary>
</details>

    
### Possible fixes

Looking at the [code](https://github.com/tiredofit/docker-db-backup/blob/fb3b65b33a610cb3cf0cae24d61b6add465bec56/install/assets/functions/10-db-backup#L706-L763), this seems to effect all backup locations as they use the `${checksum_extension}` variable, which is only set when `ENABLE_CHECKSUM` is `true` (see [`generate_checksum`](https://github.com/tiredofit/docker-db-backup/blob/fb3b65b33a610cb3cf0cae24d61b6add465bec56/install/assets/functions/10-db-backup#L657)).

---

> Probably adding a if var_true "${ENABLE_CHECKSUM}" ; then would suffice.

I can create a PR for this is an agreeable solution.

**Comment by tiredofit** [comment]
I think `3.9.10` will resolve this. Definitely an annoyance, but certainly a bug.

**Comment by ToshY** [comment]
Tested and works.

**Comment by pimjansen** [comment]
Im still getting this error though:

```
[tiredofit/db-backup:3.9.11 09:06:28 /] $ backup-now
** Performing Manual Backup
2023-09-04.09:06:30 [NOTICE] ** [db-backup] Dumping MySQL/MariaDB database: 'mydb' and compressing with 'gzip'
2023-09-04.09:06:50 [INFO] ** [db-backup] DB Backup of 'mysql_001.mysql.database.azure.com_20230904-090630.sql.gz' completed successfully
2023-09-04.09:06:50 [NOTICE] ** [db-backup] Backup of mysql_001.mysql.database.azure.com_20230904-090630.sql.gz created with the size of 8663509 bytes
2023-09-04.09:06:50 [INFO] ** [db-backup] Moving backup to S3 Bucket with blobxfer
mv: can't rename '/tmp/backups/*.': No such file or directory
2023-09-04.09:06:51 [NOTICE] ** [db-backup] DB Backup for 'oerknal_test1_firstklas' time taken: Hours: 0 Minutes: 00 Seconds: 21
2023-09-04.09:06:51 [INFO] ** [db-backup] Backup routines finish time: 2023-09-04 09:06:51 CEST with overall exit code 0
2023-09-04.09:06:51 [NOTICE] ** [db-backup] Backup routines time taken: Hours: 0 Minutes: 00 Seconds: 21
2023-09-04.09:06:51 [INFO] ** [db-backup] Cleaning up old backups on filesystem
2023-09-04.09:06:51 [INFO] ** [db-backup] Syncing changes via blobxfer
[tiredofit/db-backup:3.9.11 09:06:51 /] $ 
```

**Comment by ToshY** [comment]
Seems that @tiredofit fixed it for cases `file` / `filesystem` in [this commit](https://github.com/tiredofit/docker-db-backup/compare/3.9.9...3.9.10#diff-5e2954f9abcfee9f7dd93672770d3e518c463b3ad77160dcfed2c6b64b22f2d6R710), but not for [`s3` / `minio`](https://github.com/tiredofit/docker-db-backup/blob/main/install/assets/functions/10-db-backup#L747) or [`blobxfer`](https://github.com/tiredofit/docker-db-backup/blob/main/install/assets/functions/10-db-backup#L754-L760).

**Comment by ToshY** [comment]
@pimjansen seems fixed in [`3.9.12`](https://github.com/tiredofit/docker-db-backup/releases/tag/3.9.12).

Thanks @tiredofit 🥇

**Comment by pimjansen** [comment]
@ToshY im sure you can bump that right?
